test/test.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
Don't disable this rule. It protects you against things considered as bad practices.

Instead of ```isNaN```, you can check the ```typeof``` response and see if it equals 'number'.